x-pack/plugins/triggers_actions_ui/public/application/hooks/constants.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const triggersActionsUiQueriesKeys = {
```

Comment:
I followed the same pattern we do in Cases. Inspired by this article https://tkdodo.eu/blog/effective-react-query-keys#use-query-key-factories